src/main/java/com/softserve/academy/spaced/repetition/domain/Account.java

```diff
@@ -39,6 +38,10 @@
             inverseJoinColumns = {@JoinColumn(name = "authority_id")})
     private Set<Authority> authorities;
 
+    @NotNull
+    @Column(name = "learning_regime", columnDefinition = "int default 1")
```

Comment:
Please inroduce an enum. See http://docs.oracle.com/javaee/6/api/javax/persistence/Enumerated.html for more details.

Reply:
Can I take it as a separate task in order not to make this one too much complicated? I mean the whole system of changing the learning regime, because it can be more complex as it seems to be. For example, we might need to add the whole new entity called learning regime which could contain an id, a name and a descriptiption.

Reply:
I think, no. It is a very bad practice to write something like

if (user.getAccount().getLearningRegime() == 1) {

It isn't too much to change if you use EnumType.ORDINAL

Reply:
Oh, that is what you meant. I suppose, I've caught the whole idea, I'm going to change it.